public/xmlns/article.xsd

```diff
@@ -1,4 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
-  <xs:element name="item"/>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           attributeFormDefault="unqualified" elementFormDefault="qualified"
+           targetNamespace="https://mobile-content-api.cru.org/xmlns/article">
+  <xs:element name="article">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element name="category" type="xs:string" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
Each of these elements will need to be translated manually according to the current spec